code/modules/projectiles/guns/special/ripper/ripper.dm

```diff
@@ -27,7 +27,7 @@
 
 
 
-	has_safety = FALSE	//Safety switches are for military/police weapons, not for tools
```

Comment:
thats not how safety works. Power tools don't typically have a switch that prevents them from working, they have a dead man's switch that makes it only operate while the trigger is held

The concept of a thing that prevents firing is a thing for firearms, not for tools. Please revert this change

Reply:
@NanakoAC Citing real life, most power tools that involve drills, large saws and blades do have safeties. I'd be shocked if something of the Ripper's caliber didn't have something to prevent it from misfiring.

Reply:
Drills, large saws, blades? I have electrical screwdriver that has safety.